workspaces/app/src/foundation/components/Text.tsx

```
@@ -46,9 +47,8 @@ export const Text: React.FC<Props> = ({
   useEffect(() => {
     (async () => {
       if (!src) return;
-      const response = await fetch(src);
-      if (!response.ok) throw new Error(response.statusText);
-      setText(<>{await response.text()}</>);
+      const response = await apiClient.get(src);
+      setText(<>{response.data}</>);
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)